src/libraries/System.Private.CoreLib/src/System/Math.cs

```diff
@@ -1299,6 +1299,18 @@ public static double Round(double value, int digits)
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public static double Round(double value, MidpointRounding mode)
         {
+            // Inline single-instruction modes
+            if (RuntimeHelpers.IsKnownConstant((int)mode))
+            {
+                if (mode == MidpointRounding.ToEven)
+                    return Round(value);
+
+                // For ARM/ARM64 we can lower it down to a single instruction FRINTA
+                // For XARCH we have to use the common path
+                if (AdvSimd.Arm64.IsSupported && mode == MidpointRounding.AwayFromZero)
+                    return AdvSimd.Arm64.RoundAwayFromZero(Vector128.CreateScalarUnsafe(value)).ToScalar();
```

Comment:
This should be `AdvSimd.RoundAwayFromZeroScalar`

Reply:
Thank you